### PR TITLE
docs: Add information regarding updating extensions

### LINKED ISSUE #8377: More information updating podman desktop extensions on release doc
### Is your enhancement related to a problem? Please describe

Add more information on release doc regarding updating the podman desktop extensions

### Describe the solution you'd like

Release doc update

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by cdrage**
docs: Add information regarding updating extensions

### What does this PR do?

Adds more information to the release documentation regarding updating
extensions.

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

Closes https://github.com/containers/podman-desktop/issues/8377

### How to test this PR?

N/A

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

**Comment by deboer-tim** [comment]
I would prefer not to do this:
 - Downstream extension developers can already watch GitHub or get release notifications; if we ever need to broadcast something like a pre-release warning or technical changes, using an opt-in mailing list or discord channel would be far easier to manage.
 - There are always going to be cases where a change in Podman Desktop might affect or be interesting to lots of extensions (e.g. light mode, new API, potential breaking change), but in general we should aim to avoid stacked releases, e.g. by managing API evolution and allowing the same extension to continue working with newer Podman Desktop releases. Even with something like light mode it doesn't affect any extension that doesn't have UI, being in sync doesn't _actually_ matter, and we could have done more to allow extensions to release first before we flipped the switch.

File: RELEASE.md [hunk 1]
```diff
@@ -152,6 +152,27 @@ You can view the progress at: https://community.chocolatey.org/packages/podman-d
 1. Create a PR to the repository with a title like `feat: bump to v0.11.0`
 1. If the PR passes all tests, merge the PR
 
+## Updating Podman Desktop associated extensions
+
+There are numerous tightly integrated Podman Desktop integrations that should receive notifications that a new release has gone out.
```
Comment (lstocchi) [inline RELEASE.md:157]: Maybe extensions?
```suggestion
There are numerous tightly integrated Podman Desktop extensions that should receive notifications that a new release has gone out.
```

**Review by lstocchi — APPROVED** [review]

**Comment by lstocchi** [comment]
Ops, sorry. I didn't see Tim's comment before approving.

**Comment by cdrage** [comment]
> I would prefer not to do this:
> 
> * Downstream extension developers can already watch GitHub or get release notifications; if we ever need to broadcast something like a pre-release warning or technical changes, using an opt-in mailing list or discord channel would be far easier to manage.
> * There are always going to be cases where a change in Podman Desktop might affect or be interesting to lots of extensions (e.g. light mode, new API, potential breaking change), but in general we should aim to avoid stacked releases, e.g. by managing API evolution and allowing the same extension to continue working with newer Podman Desktop releases. Even with something like light mode it doesn't affect any extension that doesn't have UI, being in sync doesn't _actually_ matter, and we could have done more to allow extensions to release first before we flipped the switch.

I do agree, I feel as though it'd be annoyance if we'd open issues / ping extension developers at a time.

I feel as though if it is a breaking change / something critical we would notify them.

We can have this an informal procedure that we should inform extensions (and the extensions we develop) that there is an update.

**Pull request closed without merging**